src/api.v2/controllers/cellLevelMetaController.js

```diff
@@ -44,7 +45,27 @@ const updateCellLevelMetadata = async (req, res) => {
   await new CellLevelMeta().updateById(cellMetadataFileId, snakeCasedKeysToPatch);
   res.json(OK());
 };
+
+const downloadCellLevelFile = async (req, res) => {
```

Comment:
please be consistent with naming. As before, either use `cellLevelMeta` or `cellLevelMetadata` in the name. Pick one of them and use it everywhere

Reply:
Or, since this function is part of `cellLevelMetaController`, you can decide to not include cell level metadata in the name and just call it `deleteFile`. It is also a viable option. If you do that, please update all function names in this controller.